### ISSUE TITLE
Since 1.3.0 the exclude files config option doesn't work properly

### ISSUE THREAD

**Issue by rooby** [rstackjs/ts-checker-rspack-plugin#73]
## Current behavior
Given this config as an example:
```
new TsCheckerRspackPlugin({
  issue: {
    include: [{ file: 'src/my/dir/**/*.{ts,tsx}' }],
    exclude: [{ file: 'src/my/dir/temp/*.ts' }],
  },
});
```
where you exclude some files within an included dir.

In 1.3.0 if I don't have any exclude I get some errors in multiple files.
Then if I add one of the problem files to the exclude list it then passes, even though there are still errors in the other files.

## Expected behavior
Regardless of what is in the exclude files list, errors in included files should be caught.
This worked properly in 1.2.6 and earlier versions, and is one of the advantages of this project over using tsc directly.

## Steps to reproduce the issue
- Have a directory with at least 2 files.
- Have errors in both files.
- Have that directory in the include config.
- Have one of the files in the exclude config.
- The build will pass when it should fail.

## Environment
 - **ts-checker-rspack-plugin**: 1.3.0
 - **typescript**: 4.9.5
 - **@rspack/core**: 1.4.10
 - **os**: macOS 26.2
